client-app/pages/account/order-details.vue

```diff
@@ -0,0 +1,149 @@
+              <p><span class="font-extrabold">Email:</span> {{ order?.inPayments?.[0]?.billingAddress?.email }}</p>
+            </div>
+          </VcCard>
+          <VcCard title="Shipping method" :is-collapsible="true" class="mb-5">
+            <div class="flex items-center space-x-4 text-sm">
+              <VcImage src="/static/images/checkout/fedex.svg" class="h-12 w-12" />
+              <span
+                >{{ order?.shipments?.[0]?.shipmentMethodCode }} {{ order?.shipments?.[0]?.shipmentMethodOption }} ({{
+                  order?.shipments?.[0]?.price?.formattedAmount
+                }})</span
+              >
+            </div>
+          </VcCard>
+          <VcCard title="Shipping address" :is-collapsible="true" class="mb-5">
+            <div class="flex flex-col text-sm">
+              <span class="font-extrabold"
+                >{{ order?.shipments?.[0]?.deliveryAddress?.firstName }}
+                {{ order?.shipments?.[0]?.deliveryAddress?.lastName }}</span
+              >
+              <p>
+                {{ order?.shipments?.[0]?.deliveryAddress?.countryCode }}
+                {{ order?.shipments?.[0]?.deliveryAddress?.regionName }}
+                {{ order?.shipments?.[0]?.deliveryAddress?.city }}
+                {{ order?.shipments?.[0]?.deliveryAddress?.line1 }}
+                {{ order?.shipments?.[0]?.deliveryAddress?.postalCode }}
+              </p>
+              <p><span class="font-extrabold">Phone:</span> {{ order?.shipments?.[0]?.deliveryAddress?.phone }}</p>
+              <p><span class="font-extrabold">Email:</span> {{ order?.shipments?.[0]?.deliveryAddress?.email }}</p>
+            </div>
+          </VcCard>
+          <VcCard title="Payment details" :is-collapsible="true" class="mb-5">
+            <div class="flex flex-col text-sm">
+              <p><span class="font-extrabold">Payment #:</span> {{ order?.inPayments?.[0]?.number }}</p>
+              <p><span class="font-extrabold">Payment type:</span> {{ order?.inPayments?.[0]?.gatewayCode }}</p>
+              <div class="mt-3">
+                <VcButton class="px-2 py-1 uppercase text-xs" size="xs" is-outline :is-disabled="true"
+                  >View invoice</VcButton
+                >
+              </div>
+            </div>
+          </VcCard>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { CheckoutSection, OrderSummary, ProductCard } from "@/shared/checkout";
+import { CustomerOrderType } from "@/core/api/graphql/types";
+import { computed, onMounted, PropType, Ref, ref } from "vue";
+import { VcCard, VcImage, VcPagination, VcButton } from "@/components";
+import { getMyOrder } from "@/core/api/graphql/account";
+import { useRoute } from "vue-router";
+import moment from "moment";
+
+const route = useRoute();
+
+const order: Ref<CustomerOrderType | null> = ref(null);
+const orderId = ref(route.params.id as string);
+
+const itemsPerPage = ref(4);
```

Comment:
"Magic numbers" should be defined as constants in UPPERCASE in the top of the script:
```
const MAX_ITEMS = 4;
...
const itemsPerPage = ref(MAX_ITEMS);
```